src/util/TypingUtil.js

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) 2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+define(['vendor/TypingDNA'], function (TypingDNA) {
+  var tdna;
+  return {
+    track: function(selector) {
```

Comment:
nit: since this works with an element `id`, maybe we should name the argument `selectorId`